workspace/src/app/views/taskViews/linking/activeLearning/config/ManualComparisonPairSelection.tsx

```diff
@@ -313,6 +313,7 @@ const PathAutoCompletion = ({
         <ComparisonDataCell className="diapp-linking-learningdata__pathselection">
             {resetting ? null : (
                 <AutoSuggestion
```

Comment:
`AutoSuggestion` is deprecated, pls use the new component name `CodeAutocompleteField`.